packages/terra-clinical-data-grid/src/DataGrid.jsx

```diff
@@ -296,6 +314,9 @@ class DataGrid extends React.Component {
     document.addEventListener('keydown', this.handleKeyDown);
     document.addEventListener('keyup', this.handleKeyUp);
 
+    this.getLabelText();
+    this.getDescriptionText();
```

Comment:
Might be an edge case, but if the element either of these refers to changes, will the accessible text update as well?

Reply:
The way it is implemented currently it shouldn't update based on those props values changing because it is only setting state when this component mounts. To do that, we'd probably want to add these calls to the [`componentDidUpdate`](https://react.dev/reference/react/Component#componentdidupdate) lifecycle method.

Reply:
![UpdateLabelDesc](https://user-images.githubusercontent.com/76790350/231016425-d3095797-ddbe-45f0-9ceb-b01a26d50057.gif)

Was able to get this to update the hidden text (see spans in gif) using componentDidUpdate. Will be in my next commit